packages/clay-css/src/images/icons/search.svg

```diff
@@ -1,3 +1,10 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
-	<path class="lexicon-icon-outline" d="M503.254 467.861l-133.645-133.645c27.671-35.13 44.344-79.327 44.344-127.415 0-113.784-92.578-206.362-206.362-206.362s-206.362 92.578-206.362 206.362 92.578 206.362 206.362 206.362c47.268 0 90.735-16.146 125.572-42.969l133.851 133.851c5.002 5.002 11.554 7.488 18.106 7.488s13.104-2.486 18.106-7.488c10.004-10.003 10.004-26.209 0.029-36.183zM52.446 206.801c0-85.558 69.616-155.173 155.173-155.173s155.174 69.616 155.174 155.173-69.616 155.173-155.173 155.173-155.173-69.616-155.173-155.173z"></path>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@kresimir-coko We can reduce the svg source to:
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
	<path class="lexicon-icon-outline" d="M499.2,455.5L377.7,333.4c146-241.1-148.1-435.8-318.2-274c-165.1,185.9,41.6,460.6,273.4,319l121.5,118.8C489.5,535.8,534.4,490.8,499.2,455.5z M206.2,63.6c191.9,0,198.1,289,0,289C13.3,352.6,18.8,63.6,206.2,63.6z"/>
</svg>
```
We should add the class `lexicon-icon-outline` on any SVG element (e.g., `path`, `circle`, `rect`, ect) that should inherit color.

We shouldn't include any styles in the SVG because it makes it harder to overwrite.

For more complex icons that have several elements, I've been adding classes `lx-{the icon name}-{a semantic name for the element}` (e.g., `lx-briefcase-bottom`). This makes it easier to change a specific element inside the svg via CSS.